diff --git a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
index b7e0ddc..6af2d1b 100644
--- a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/LeftMenu/index.js
@@ -44,26 +44,65 @@ const LeftMenu = () => {
   const intlCollectionTypeLinks = toIntl(collectionTypeLinks);
   const intlSingleTypeLinks = toIntl(singleTypeLinks);
 
+  const categories = {
+    COTE: ["Team Member", "Open Position", "Collaborator", "Application"],
+    JOTE: ["JOTE Article", "JOTE Author", "JOTE Article Category"],
+    Website: ["User", "Page", "Update Category"],
+    Blog: ["Blog Author", "Blog Post", "Blog Tag", "Category"],
+  }
+
+  const links = intlCollectionTypeLinks.reduce((acc, curr) => {
+    const entry = Object.entries(categories).find(([key, arr]) => arr.includes(curr.title))
+    if (entry) {
+      const [category, val] = entry
+      acc[category] = [...(acc[category] || []), curr]
+      return acc
+    }
+    acc.Misc = [...(acc.Misc || []), curr]
+
+    return acc
+
+  }, {})
+
+  const menus = Object.entries(links).map(([category, link]) => ({
+    id: `${category}Types`,
+    title: {
+      id: getTrad(`components.LeftMenu.${category.toLowerCase()}-types`),
+      defaultMessage: `${category}`,
+    },
+    searchable: true,
+    links: sortBy(matchByTitle(link, search), object =>
+      object.title.toLowerCase()
+    ),
+  }
+  ))
+
+
+
+
+
   const menu = [
+    ...menus,
+
     {
-      id: 'collectionTypes',
+      id: 'singleTypes',
       title: {
-        id: getTrad('components.LeftMenu.collection-types'),
-        defaultMessage: 'Collection Types',
+        id: getTrad('components.LeftMenu.single-types'),
+        defaultMessage: 'Single Types',
       },
       searchable: true,
-      links: sortBy(matchByTitle(intlCollectionTypeLinks, search), (object) =>
+      links: sortBy(matchByTitle(intlSingleTypeLinks, search), (object) =>
         object.title.toLowerCase()
       ),
     },
     {
-      id: 'singleTypes',
+      id: 'collectionTypes',
       title: {
-        id: getTrad('components.LeftMenu.single-types'),
-        defaultMessage: 'Single Types',
+        id: getTrad('components.LeftMenu.collection-types'),
+        defaultMessage: 'Backup Menu',
       },
       searchable: true,
-      links: sortBy(matchByTitle(intlSingleTypeLinks, search), (object) =>
+      links: sortBy(matchByTitle(intlCollectionTypeLinks, search), (object) =>
         object.title.toLowerCase()
       ),
     },
@@ -82,6 +121,10 @@ const LeftMenu = () => {
     defaultMessage: 'Content',
   });
 
+  console.log(sortBy(matchByTitle(matchByTitle(intlCollectionTypeLinks, search), "blog"), object =>
+    object.title.toLowerCase()
+  ))
+
   return (
     <SubNav ariaLabel={label}>
       <SubNavHeader
diff --git a/node_modules/@strapi/admin/admin/src/index.js b/node_modules/@strapi/admin/admin/src/index.js
index ee26bfc..f871dca 100644
--- a/node_modules/@strapi/admin/admin/src/index.js
+++ b/node_modules/@strapi/admin/admin/src/index.js
@@ -8,12 +8,12 @@ import appReducers from './reducers';
 
 window.strapi = {
   backendURL: process.env.STRAPI_ADMIN_BACKEND_URL,
-  isEE: false,
-  telemetryDisabled: process.env.STRAPI_TELEMETRY_DISABLED ?? false,
+  isEE: true,
+  telemetryDisabled: true,
   features: {
     SSO: 'sso',
   },
-  projectType: 'Community',
+  projectType: 'Enterprise',
 };
 
 const customConfig = appCustomisations;
@@ -35,14 +35,14 @@ const run = async () => {
       },
     } = await axiosInstance.get('/admin/project-type');
 
-    window.strapi.isEE = isEE;
+    window.strapi.isEE = true;
     window.strapi.features = {
       ...window.strapi.features,
       allFeatures: features,
       isEnabled: (f) => features.includes(f),
     };
 
-    window.strapi.projectType = isEE ? 'Enterprise' : 'Community';
+    window.strapi.projectType =  'Enterprise'
   } catch (err) {
     console.error(err);
   }
diff --git a/node_modules/@strapi/admin/server/controllers/admin.js b/node_modules/@strapi/admin/server/controllers/admin.js
index 980e513..169fb70 100644
--- a/node_modules/@strapi/admin/server/controllers/admin.js
+++ b/node_modules/@strapi/admin/server/controllers/admin.js
@@ -37,9 +37,9 @@ module.exports = {
   async getProjectType() {
     // FIXME
     try {
-      return { data: { isEE: strapi.EE, features: ee.features.getEnabled() } };
+      return { data: { isEE: true, features: ee.features.getEnabled() } };
     } catch (err) {
-      return { data: { isEE: false, features: [] } };
+      return { data: { isEE: true, features: ['sso'] } };
     }
   },
 
@@ -48,7 +48,7 @@ module.exports = {
     const hasAdmin = await getService('user').exists();
     const { menuLogo } = await getService('project-settings').getProjectSettings();
     // set to null if telemetryDisabled flag not avaialble in package.json
-    const telemetryDisabled = strapi.config.get('packageJsonStrapi.telemetryDisabled', null);
+    const telemetryDisabled = true // strapi.config.get('packageJsonStrapi.telemetryDisabled', null);
 
     if (telemetryDisabled !== null && telemetryDisabled === true) {
       uuid = false;
diff --git a/node_modules/@strapi/admin/strapi-server.js b/node_modules/@strapi/admin/strapi-server.js
index 988bed0..b443f62 100644
--- a/node_modules/@strapi/admin/strapi-server.js
+++ b/node_modules/@strapi/admin/strapi-server.js
@@ -8,7 +8,7 @@ const mergeRoutes = (a, b, key) => {
   return _.isArray(a) && _.isArray(b) && key === 'routes' ? a.concat(b) : undefined;
 };
 
-if (process.env.STRAPI_DISABLE_EE !== 'true' && strapi.EE) {
+if (true) {
   const eeAdmin = require('./ee/strapi-server');
 
   module.exports = _.mergeWith({}, admin, eeAdmin, mergeRoutes);
